app/store/transaction/transaction.reducer.ts

```diff
@@ -19,7 +19,10 @@ export interface TransactionState extends EntityState<AddressTransactionWithTran
 
 const transactionAdapter = createEntityAdapter<AddressTransactionWithTransfers>({
   selectId: ({ tx }) => tx.tx_id,
-  sortComparer: (tx1, tx2) => tx2.tx.burn_block_time - tx1.tx.burn_block_time,
+  sortComparer: (tx1, tx2) => {
+    if (tx1.tx.burn_block_time_iso === '') return -1;
```

Comment:
Can also use is_unanchored here, I imagine

Reply:
Good catch ! I will fix it.